service-front/module/Application/view/application/pages/driving_licence_number_success.twig

```diff
@@ -19,18 +19,16 @@
             Read the following to the caller:
         </h2>
         <p class="govuk-body">
-            Thank you - we have confirmed your identity document is valid.
+            Thank you - your initial identity check is complete and your identity verified.
```

Comment:
from the design on figma it looks like the text is meant to be [inset](https://design-system.service.gov.uk/components/inset-text/)